src/platform/forms-system/src/js/helpers.js

```diff
@@ -16,9 +16,9 @@ export const maxYear = moment()
 // An active page is one that will be shown to the user.
 // Pages become inactive if they are conditionally shown based
 // on answers to previous questions.
-export function isActivePage(page, data) {
+export function isActivePage(page, data, isLoggedIn) {
   if (typeof page.depends === 'function') {
-    return page.depends(data, page.index);
+    return page.depends(data, page.index, isLoggedIn);
```

Comment:
It'd be nice to pass an object as the third parameter... personally, I'd like to see `featureToggles` also passed to the depends function, so it could be done in this work, or in the future without adding a whole bunch of parameters

Reply:
Yeah, I agree with Robin. Instead of passing just a boolean, we may want to pass an `options` object, or even `formState`.

Reply:
I'm open to passing whatever we want, I just had `isLoggedIn` in mind because it fit my use case